src/main/sensors/battery.c

```diff
@@ -84,9 +83,9 @@ static void updateBatteryVoltage(void)
         vBatFilterIsInitialised = true;
     }
 
-    #ifdef USE_ESC_TELEMETRY
-    if (batteryConfig->batteryMeterType == BATTERY_SENSOR_ESC && isEscTelemetryActive()) {
-        vbatLatest = getEscTelemetryVbat();
+    #ifdef USE_ESC_SENSOR
+    if (feature(FEATURE_ESC_SENSOR) && batteryConfig->batteryMeterType == BATTERY_SENSOR_ESC) {
```

Comment:
Yay, thanks for this.

I think the voltage meter setting should be added to the end of `MSP_VOLTAGE_METER_CONFIG` / `MSP_SET_VOLTAGE_METER_CONFIG`.